graphql-router/src/main/java/com/networknt/graphql/router/handlers/GraphqlPostHandler.java

```diff
@@ -48,6 +53,18 @@
             logger.error("Unable to load GraphQL schema - no SchemaProvider implementation in service.yml");
             throw new RuntimeException("Unable to load GraphQL schema - no SchemaProvider implementation in service.yml");
         }
+
+        // Replace default execution strategies if so configured.
+        ExecutionStrategyProvider executionStrategyProvider = SingletonServiceFactory.getBean(ExecutionStrategyProvider.class);
+        if(executionStrategyProvider != null) {
```

Comment:
In the three condition checks, i think you meant to include an additional condition for the result of the get methods, right? Otherwise it would be weird to check the same condition 3 separate times..

Reply:
Uh, yes. This is straight up broken code. Let me redo it.